sycl/doc/extensions/experimental/sycl_ext_oneapi_device_image_backend_content.asciidoc

```diff
@@ -101,7 +99,7 @@ class device_image {
   backend ext_oneapi_get_backend() const noexcept;
   std::vector<std::byte> ext_oneapi_get_backend_content() const;
 
-  std::span<std::byte> ext_oneapi_get_backend_content_view() const; // Requires C++20
+  std::span<const std::byte> ext_oneapi_get_backend_content_view() const; // Requires C++20
```

Comment:
I know the spec says `std::span` and goes on to note that this requires C++20.

But it might be worth noting that we do have `sycl::span` which is available even if not using C++20.

Reply:
Thanks Chris!
@gmlueck FYI

Reply:
`sycl::span` and `std::span` are different types.  Therefore, we would need two separate functions: one that returns `std::span` and another that returns `sycl::span`.  I considered doing this, but it did not seem worth the effort and additional maintenance cost.  If an application wants the more efficient API, I think it's reasonable to ask them to compile in C++20.